### ISSUE TITLE
Question - data with unit length

### ISSUE THREAD

**Issue by daniellawton99** [idstcv/SoftTriple#15]
The paper mentiones the data (x) is of unit length. It even seems to be a requirement for using SoftTriplet. 
Is this correct? How did you guarantee this in your experiments?

Thanks!

**Comment by qian-qi** [comment]
Hi,
The normalization is in Line 493 of model file (SoftTriple/net/bninception.py) as "x = F.normalize(x, p=2, dim=1)".
